test/intl402/DateTimeFormat/prototype/formatRangeToParts/en-US.js

```diff
@@ -0,0 +1,199 @@
+// Copyright (C) 2019 the V8 project authors, Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-partitiondatetimerangepattern
+description: Basic tests for the en-US output of formatRangeToParts()
+info: >
+  Intl.DateTimeFormat.prototype.formatRangeToParts ( startDate , endDate )
+
+  8. Return ? FormatDateTimeRange(dtf, x, y).
+locale: [en-US]
+features: [Intl.DateTimeFormat-formatRange]
+---*/
+
+function* zip(a, b) {
+  assert.sameValue(a.length, b.length);
+  for (let i = 0; i < a.length; ++i) {
+    yield [i, a[i], b[i]];
+  }
+}
+
+function compare(actual, expected) {
+  for (const [i, actualEntry, expectedEntry] of zip(actual, expected)) {
+    assert.sameValue(actualEntry.type, expectedEntry.type, `type for entry ${i}`);
+    assert.sameValue(actualEntry.value, expectedEntry.value, `value for entry ${i}`);
+    assert.sameValue(actualEntry.source, expectedEntry.source, `source for entry ${i}`);
+  }
+}
+
+const date1 = new Date("2019-01-03");
+const date2 = new Date("2019-01-05");
+const date3 = new Date("2019-03-04");
+const date4 = new Date("2020-03-04");
```

Comment:
I believe these need to be
```
const date1 = new Date("2019-01-03T00:00");
const date2 = new Date("2019-01-05T00:00");
const date3 = new Date("2019-03-04T00:00");
const date4 = new Date("2020-03-04T00:00");
```
to make it clear that date is in local time. Otherwise, if we use new Date("2019-01-03"); on a Machine in PST. The date object is in UTC time but the formatted result will be in local time zone. 

This is because in 
https://ecma-international.org/ecma-262/#sec-date-time-string-format
it stated:
```
... When the time zone offset is absent, date-only forms are interpreted as a UTC time and date-time forms 
 are interpreted as a local time."
```